src/api/app/models/buildresult.rb

```diff
@@ -85,6 +85,7 @@ def self.summary(project_name)
         state: state
       )
 
+      StatusCount = Struct.new(:code, :count)
```

Comment:
To make Rubocop happy: `Struct.new('StatusCount', :code, :count)`, but I think this could be a hash too. The lines with `build.summary` would have to be changed too if we do that.

Reply:
thanks!